src/KOKKOS/atom_kokkos.cpp

```diff
@@ -318,7 +331,11 @@ int AtomKokkos::add_custom(const char *name, int flag, int cols, int ghost)
     iaghost = (int *) memory->srealloc(iaghost, niarray * sizeof(int), "atom:iaghost");
     iaghost[index] = ghost;
     iarray = (int ***) memory->srealloc(iarray, niarray * sizeof(int **), "atom:iarray");
-    memory->create(iarray[index], nmax, cols, "atom:iarray");
+    auto name = fmt::format("atom:iarray[{}]", index);
+    k_iarray.emplace_back(name, nmax, cols);
+    this->sync(Device, IARRAY_MASK);
+    memoryKK->grow_kokkos(k_iarray[index], iarray[index], nmax, cols, name.c_str());
```

Comment:
@stanmoore1 this is the part I mentioned to you, since I'm creating a new dual view here with `emplace_back`, but then immediately throwing it away in `grow_kokkos`.